priv/perf/apps/load_test/lib/scenario/deposits.ex

```diff
@@ -0,0 +1,159 @@
+
+    session
+    |> cc_spread(
+      :create_deposit_and_make_assertions,
+      total_number_of_transactions,
+      period_in_mseconds
+    )
+    |> await_all(:create_deposit_and_make_assertions)
+  end
+
+  def create_deposit_and_make_assertions(session) do
+    with {:ok, from, to} <- create_accounts(session),
+         :ok <- create_deposit(from, session),
+         :ok <- send_value_to_receiver(from, to, session) do
+      Session.add_metric(session, "success_rate", 1)
+    else
+      error ->
+        log_error(session, "#{__MODULE__} failed with #{inspect(error)}")
+
+        session
+        |> Session.add_metric("success_rate", 0)
+        |> Session.add_error(:test, error)
+    end
+  end
+
+  defp create_accounts(session) do
+    initial_amount = config(session, [:chain_config, :initial_amount])
+    {:ok, from_address} = Account.new()
+    {:ok, to_address} = Account.new()
+
+    {:ok, _} = Faucet.fund_root_chain_account(from_address.addr, initial_amount)
+
+    {:ok, from_address, to_address}
+  end
+
+  defp create_deposit(from_address, session) do
+    token = config(session, [:chain_config, :token])
+    deposited_amount = config(session, [:chain_config, :deposited_amount])
+    initial_amount = config(session, [:chain_config, :initial_amount])
+
+    txhash =
+      Deposit.deposit_from(from_address, deposited_amount, token,
+        return: :txhash,
+        deposit_finality_margin: 10,
+        gas_price: 0
+      )
+
+    gas_used = Ethereum.get_gas_used(txhash)
+
+    with :ok <-
+           fetch_childchain_balance(from_address,
+             amount: deposited_amount,
+             token: token,
+             error: :wrong_childchain_from_balance_after_deposit
+           ),
+         :ok <-
+           fetch_rootchain_balance(
+             from_address,
+             amount: initial_amount - deposited_amount - gas_used,
+             token: token,
+             error: :wrong_rootchain_balance_after_deposit
+           ) do
+      :ok
+    else
+      error ->
+        error
+    end
+  end
+
+  defp send_value_to_receiver(from_address, to_address, session) do
+    token = config(session, [:chain_config, :token])
+    transferred_amount = config(session, [:chain_config, :transferred_amount])
+
+    with _ <- send_amount_on_childchain(from_address, to_address, token, transferred_amount),
+         :ok <-
+           fetch_childchain_balance(
+             to_address,
+             amount: transferred_amount,
+             token: token,
+             error: :wrong_childchain_to_balance_after_sending_deposit
+           ) do
+      :ok
+    else
+      error -> error
+    end
+  end
+
+  defp send_amount_on_childchain(from, to, token, amount) do
+    {:ok, [sign_hash, typed_data, _txbytes]} =
+      Ethereum.create_transaction(
+        amount,
+        from.addr,
+        to.addr,
+        token
+      )
+
+    _ = Ethereum.submit_transaction(typed_data, sign_hash, [from.priv])
```

Comment:
this doesn't do anything
```suggestion
   Ethereum.submit_transaction(typed_data, sign_hash, [from.priv])
```